.github/dependabot.yml

```diff
@@ -11,3 +11,4 @@ updates:
   - gautam-yoti
   assignees:
   - echarrod
```

Comment:
We try to usually have one "assignee", so that one person is responsible for merging:
```suggestion
```